v0.2.8-rc.0

chepix10 opened the pull request:
Added support for descriptors
Added support for businessUuid
Added tests for descriptors and businessUuid
Updated documentation methods for redeem, issue and transfer
Bumped version to 0.2.8-rc.0

amack300 commented:
Hey @chepix10, I have yet to test the code changes just yet, but I did notice that the "transaction_id" property is missing from the response.    Is that something you can add it at your earliest convenience.  Thanks!

Per the docs:  https://docs.silamoney.com/?plaintext#issue_sila

```
HTTP/1.1 200 OK

{
  "reference": "ref",
  "message": "Transaction submitted to the processing queue.",
  "success": true,
  **"transaction_id": "UUID of the submitted transaction",**
  "descriptor": "optional transaction descriptor"
}
```

chepix10 commented:
Hi @amack300. Thanks for your feedback. The field is being sent on the response object. I've updated the documentation to include the transaction_id field. I've also added the field to the integration tests.
